internal/fs/fs.go

```diff
@@ -1972,6 +1972,45 @@ func (fs *fileSystem) renameFile(
 	return nil
 }
 
+func (fs *fileSystem) handlePendingInodes(inodes *[]inode.DirInode) func() {
```

Comment:
why is this function returning a function?

Reply:
Done